tests/unit/test_oneview_client.py

```diff
@@ -930,3 +931,10 @@ def test_hypervisor_managers_has_right_type(self):
     def test_lazy_loading_hypervisor_managers(self):
         hypervisor_managers = self._oneview.hypervisor_managers
         self.assertNotEqual(hypervisor_managers, self._oneview.hypervisor_managers)
+
+    def test_certificates_server_has_right_type(self):
+        self.assertIsInstance(self._oneview.certificates_server, CertificatesServer)
+
+    def test_lazy_loading_certificates_server(self):
```

Comment:
We are not using lazy loading for the resources in the new design. update the test case name

Reply:
Done